Makefile

```diff
@@ -0,0 +1,25 @@
+docstyle:
```

Comment:
Do a `phony` target need to be added to all of these?

https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
I don't think so - regardless of whether files on disk have changed or not, make check works reliably.